root/static/scripts/common/components/DynamicAttributesList.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import bracketed from '../utility/bracketed';
+
+const DynamicAttributesList = ({entity}: {entity: WorkT}) => {
+  console.log(entity.attributes);
+  return (
+    entity.attributes ? (
+      <ul>
+        {entity.attributes.map(attribute => (
+          <li key={attribute.id}>
+            {lp_attributes(
+              attribute.value,
+              'work_attribute_type_allowed_value',
+            )}
+            {' '}
+            {bracketed(lp_attributes(
```

Comment:
You could use `bracketedText` here since it's expected to be a string (but `bracketed` will work fine too).